base/bind-utils/stone.yml

```diff
@@ -0,0 +1,54 @@
+#
+# SPDX-FileCopyrightText: © 2020-2023 Serpent OS Developers
+#
+# SPDX-License-Identifier: Zlib
+#
+name        : bind-utils
+version     : '9.18.10'
+release     : 1
+homepage    : https://www.isc.org/bind/
+upstreams   :
+    - https://downloads.isc.org/isc/bind9/9.18.10/bind-9.18.10.tar.xz : f415a92feb62568b50854a063cb231e257351f8672186d0ab031a49b3de2cac6
+summary     : Tools to interact with DNS services
+description : |
+    BIND (Berkeley Internet Name Domain) is a complete, highly portable
+    implementation of the Domain Name System (DNS) protocol. This package
+    includes client tools like dig & nslookup.
+license     :
+    - MPL-2.0-no-copyleft-exception
```

Comment:
Are you *sure* this is the right license? I think this package has different licenses for different components, you should verify this.

Reply:
I added BSD-3-Clause & MIT as well, they were used in two `*.c` files for libisc. The rest should be MPL. Although I'm a bit confused as they explicitly state `SPDX-License-Identifier: MPL-2.0` in the headers, but their `LICENSE` file matches the `MPL-2.0-no-copyleft-exception` SPDX Identifier.